src/api/app/models/token/workflow.rb

```diff
@@ -14,17 +15,21 @@ def call(options)
     yaml_file = Workflows::YAMLDownloader.new(scm_webhook.payload, token: self).call
     workflows = Workflows::YAMLToWorkflowsService.new(yaml_file: yaml_file, scm_webhook: scm_webhook, token: self).call
 
+    # FIXME: The scm_webhook has hidden objects.
```

Comment:
What do you mean exactly with `hidden objects`? Is it that `ScmWebhook` could be split in multiple classes?

Reply:
Sorry, the `Token::Workflow` object has hidden objects inside not the `scm_webhook`.

Reply:
I've updated the comment so it is clearer